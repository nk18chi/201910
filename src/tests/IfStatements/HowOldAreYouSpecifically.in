Billy_Corgan
17

