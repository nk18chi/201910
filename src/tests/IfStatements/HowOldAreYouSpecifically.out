Hey, what's your name? (Sorry, I keep forgetting.) Ok, Billy_Corgan, how old are you? 
You can drive but not vote, Billy_Corgan.
